### ISSUE TITLE
Disable/Remove/Fix MainActivity Menu

### ISSUE THREAD

**Issue by 13rac1** [roundware/roundware-android#43]
The three-dot menu on the MainActivity doesn't work right. The `Legal Notice` does nothing, and the very little of the `Preferences` actually work. We can't have enabled broken features. I think we could remove it altogether, but maybe it makes sense to have part of it? Thoughts?

**Comment by hburgund** [comment]
The reason for the Legal Notification initially was due to a requirement from Google regarding the use of Google maps, I believe.  They make you put this long annoying legal language in the app somewhere and this was the way that made the most sense.  But I have no idea if this is still a requirement.

I have never liked that little menu, but it was better than the alternatives way back when.  So I agree to remove it if possible.

**Comment by 13rac1** [comment]
Yes: https://developers.google.com/maps/documentation/android/intro#attribution_requirements

Still required.

**Comment by hburgund** [comment]
OK, that's annoying.  Any ideas how other apps handle this? I'd love to remove the menu, but not sure where else to put this legal mumbo-jumbo...?

I will add that the preferences have proven to be useful for testing and debugging, though most of them are not to be touched by an actual user, so it wasn't a good idea to include them in the public builds.
